stock_kardex_report/models/stock_kardex_report.py

```diff
@@ -0,0 +1,163 @@
+# Copyright 2020, Jarsa Sistemas, S.A. de C.V.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from odoo import _, fields, models
+import textwrap
+
+
+class StockKardexReportWiz(models.TransientModel):
```

Comment:
If it's a wizard it must be in a separate folder.. see [module structure](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#17complete-structure)

Reply:
Created folders